src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,190 @@ Jon Schneider <jschneider@pivotal.io>
+
+If a Dynatrace OneAgent is installed on the host running Micrometer, metrics can be exported directly using the OneAgent without having to specify an endpoint URI or API token.
+Use the following code in your project to export Micrometer metrics to the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/[local OneAgent endpoint]:
+
 [source,java]
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    @Nullable
+    public String get(String k) {
+        // This method for retrieving arbitrary config items introduced by the interface is currently not used in DynatraceConfig.
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+If you are working with Spring Boot, you can instead put the following properties in your `application-dynatrace.yml` in the `resources` folder.
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    enabled: true
+    api-version: v2
+----
+
+=== Using a custom endpoint
+
+If no Dynatrace OneAgent is available, both the Dynatrace Metrics API v2 endpoint and an API token have to be specified.
+The https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[Dynatrace API token documentation] contains more information on how to create an API token.
+The 'Ingest metrics' (`metrics.ingest`) permission is required on the token in order to ingest metrics.
+It is recommended to limit scope to only this permission.
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String uri() {
+        // The endpoint of the Dynatrace Metrics API v2 including path, e. g.
+        // "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest".
+        // To be read from environment variables or a config file.
+        return MY_DYNATRACE_URI;
+    }
+
     @Override
     public String apiToken() {
+        // It is *not* recommended to place secrets in config files but rather to read them from env vars, for example.
         return MY_TOKEN;
     }
 
+    @Override
+    @Nullable
+    public String get(String k) {
+        // This method for retrieving arbitrary config items introduced by the interface is currently not used in DynatraceConfig.
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+These properties can also be set via the Spring Boot properties file.
+It is possible to reference environment variables using the Spring property placeholder notation here.
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    enabled: true
```

Comment:
I don't think `enabled ` is needed. Could you please fix it in other places too?

Reply:
@jonatan-ivanov From my tests with the Micrometer Dynatrace sample application it seems as if this is needed. If I don't specify it, nothing will be exported. Am I missing a configuration option somewhere else?

Reply:
If you are referring to the DynatraceSample in the Micrometer repository, it needs to be enabled because the uber sample app contains all of the registries and hence we disable them all by default so you can run with just one registry enabled. This is mostly unique to the structure of the all-in-one sample application and not typical for a user.

Reply:
That's exactly what I'm referring to. I will remove them then - Thanks!